src/xma/include/app/xmalogger.h

```diff
@@ -28,18 +29,19 @@ extern "C" {
 /**
  * enum XmaLogLevelType - Describes the logging level associated with a log message.
 */
+
 typedef enum XmaLogLevelType
 {
-    XMA_CRITICAL_LOG = 0, /**< 0 */
-    XMA_ERROR_LOG,        /**< 1 */
-    XMA_INFO_LOG,         /**< 2 */
-    XMA_DEBUG_LOG,        /**< 3 */
+    XMA_CRITICAL_LOG = CRITICAL,
+    XMA_ERROR_LOG = ERROR,
+    XMA_INFO_LOG = INFO,
+    XMA_DEBUG_LOG = DEBUG
 } XmaLogLevelType;
 
```

Comment:
I think we should just kill XmaLogLevelType all together by remapping it with XRT message type with something like:
typedef enum XmaLogLevelType xclLogMsgLevel;
#define XMA_CRITICAL_LOG CRITICAL
#define XMA_ERROR_LOG ERROR
#define XMA_INFO_LOG INFO
#define XMA_DEBUG_LOG  DEBUG